docs/reference/commandline/swarm_join_token.md

```diff
@@ -21,13 +21,14 @@ Options:
       --rotate   Rotate join token
 ```
 
-Join tokens are secrets that determine whether or not a node will join the swarm as a manager node
-or a worker node. You pass the token using the `--token flag` when you run
-[swarm join](swarm_join.md). You can access the current tokens or rotate the tokens using
-`swarm join-token`.
+Join tokens are secrets that determine whether or not a node will join the swarm
```

Comment:
`Join tokens are secrets that allow a node to join the swarm cluster. Given that a node can either join the swarm as a manager or a worker, there are two different join tokens available.`